src/scripts/main.js

```diff
@@ -357,4 +357,26 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
+const newTh = table.tHead.rows[0].insertCell(5);
```

Comment:
These lines assume that the table has at least 5 columns in the header. Ensure that this assumption holds true in the HTML structure.